### DISCUSSION TITLE
Virtual Drivers

### DISCUSSION THREAD

**Discussion by hh2302** [MolotovCherry/virtual-display-rs#113] [General]
How do Windows Virtual Drivers pass the HLK test and get signed by Microsoft

**Comment by MolotovCherry** [comment]
To get a signing certificate, as far as I understand, you need a business, and they will check certain details of the business to make sure it's valid and meets the requirements. Then there's the certificate fee which is several hundred dollars per certificate period length (different durations can be chosen)

Don't know about HLK, never looked into it.

(Also, fwiw, I don't have to pass any of those 😄 )
